### ISSUE TITLE
Pre-trained conv layers weight

### ISSUE THREAD

**Issue by BeSlower** [hizhangp/yolo_tensorflow#8]
Could you upload a pre-trained conv layer weights (without fc layer)? Thanks.

**Comment by hizhangp** [comment]
You can add an expression `self.restorer = tf.saver(self.net.collection[:-6])` before `self.saver`, and then change `self.saver.restore(self.sess, self.weights_file)` into `self.restorer.restore(self.sess, self.weights_file)` to avoid restoring weights from *fc* layers.

**Comment by BeSlower** [comment]
Really thanks for your help.

**Comment by ChengqiBian** [comment]
I met the same problem. But adding these two expressions don't work for the recently updated version of the training code, how to restore the weight file without fc layers now? Thanks!

**Comment by hizhangp** [comment]
Just change expression `self.restorer = tf.saver(self.net.collection[:-6])` into `self.restorer = tf.saver(self.variable_to_restore[:-6])` in the new version of training code.

**Comment by CompilerBian** [comment]
Thanks for your help. I have successfully trained the model with my own dataset.

**Comment by feitiandemiaomi** [comment]
@hizhangp I understand , it is answer I need , and , if I just change the some layer ,eg 8th conv layer ,what I should modify  for train from the scratch

**Comment by ss199302** [comment]
@hizhangp  Hello, why hasn't my loss changed from the beginning to the end of training? Do you know what the reason is? Thank you very much!

**Comment by ss199302** [comment]
@hizhangp Hello, why hasn't my loss changed from the beginning to the end of training? Do you know what the reason is? Thank you very much!
